crates/rust-project-goals/src/gh/issues.rs

```diff
@@ -180,6 +199,56 @@ pub fn create_issue(
     }
 }
 
+pub fn change_milestone(
+    repository: &Repository,
+    number: u64,
+    milestone: &str,
+) -> anyhow::Result<()> {
+    let mut command = Command::new("gh");
+    command
+        .arg("-R")
+        .arg(&repository.to_string())
+        .arg("issue")
+        .arg("edit")
+        .arg(number.to_string())
+        .arg("milestone")
+        .arg(milestone);
+
+    let output = command.output()?;
+    if !output.status.success() {
+        Err(anyhow::anyhow!(
+            "failed to change milestone `{}`: {}",
+            number,
+            String::from_utf8_lossy(&output.stderr)
+        ))
+    } else {
+        Ok(())
+    }
+}
+
+pub fn create_comment(repository: &Repository, number: u64, body: &str) -> anyhow::Result<()> {
```

Comment:
the `lock_issue` function below also leaves a comment, maybe it can be refactored to use this helper?